### PR TITLE
Cookie policy

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Adds Cookie policy page based on [this file
](https://docs.google.com/document/d/1_G9YNuLxLhosWBAmNmHxgwFjeGlUCH6iDpl_IuPppYo/edit)

# To test
1. scroll down to the footer
2. click on Cookie policy link
3. you should see the cookie policy page

### CODE REVIEW

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @nenadV91 , great job!
Issues: 
1. there is a duplicating sentence: 
double text
2. In this paragraph there should be commas instead of semicolons:
comma and period
3. And it would be great to open links in new tabs
4. I don't think that the policy text should be center-aligned in a mobile view. Left aligned text looks better to me. WDYT?

Thanks!
image

**Comment by nenadV91** [comment]
@elena-zh 
1. Great catch, fixed
2. I think it is fine like that, we are doing the same on https://swap.cow.fi/#/cookie-policy?chain=mainnet
3. Unfortunately seems that we can't do this here. In cowswap we used other plugin to parse markup and we could use html in it, but here we can't do that.
4. Good suggestion, updated.

**Review by elena-zh — APPROVED** [review]
LGTM then.

File: src/const/meta.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const CONFIG = {
     cowProtocol: "https://cow.fi",
     beaver: "https://beaverbuild.org/",
     agnostic: "https://agnostic-relay.net/",
+    cookiePolicy: '/cookie-policy'
```
Comment (shoom3301) [inline src/const/meta.ts:25]: Mixed single/double quotes, I think we should use only one

**Review by shoom3301 — APPROVED** [review]

File: src/const/meta.ts (hunk 1 above)
Comment (nenadV91) [reply src/const/meta.ts]: Good catch 👍

**Pull request merged**